docs/index.html

```diff
@@ -1,47 +1,109 @@
-<!DOCTYPE HTML>
-<html lang="en">
+<!DOCTYPE html>
```

Comment:
How about using a proper github pages workflow via a second branch and the accompanying github action? That setup has been tested and is also what we use in the docs repo.

It saves us from 
1. having the same files (assets) twice in the repo,
2. forgetting to update the built files,
3. ignoring half of the changes on pull requests.

Reply:
Will work on this